COMPILING.md

```diff
@@ -1,22 +1,29 @@
+# Compiling Jamulus
+
 ## Points to note
 
-- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the binaries generated by the autobuild process in the Jamulus repository.
+- Jamulus can be compiled for Linux, Windows and macOS. However, the preferred method of supporting these platforms is to use the [binaries generated by the autobuild process](https://github.com/jamulussoftware/jamulus/releases/latest) in the Jamulus repository.
 - For unattended installs, see the contributed [installation scripts](https://github.com/jamulussoftware/installscripts)
+- The [Autobuild scripts](https://github.com/jamulussoftware/jamulus/tree/master/.github/autobuild) although only optimized to be used by the CI, might help you understand the process of setting up a build environment
 - There are reports from people who successfully compile and run Jamulus on BSDs.
 - Android and iOS are not officially supported.
 
 ---
 
 ## Download sources
 
-For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+First of all, you need to get the Jamulus source code. You can either download it manually or use git:
 
-For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.zip)
+- For .tar.gz [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.tar.gz) to download the latest release
+- For .zip [use this link](https://github.com/jamulussoftware/jamulus/archive/latest.zip)
+- If you use `git`, install it and then run `git clone git@github.com:jamulussoftware/jamulus` in Terminal to get the bleeding edge version directly from GitHub.
 
 ## Linux
 
 ### Install dependencies
 
+On Debian 11+ you can install the dependencies by issuing the following command: `sudo apt-get -qq --no-install-recommends -y install devscripts build-essential debhelper fakeroot libjack-jackd2-dev qtbase5-dev qttools5-dev-tools`
+
```

Comment:
It sounds a bit strange to list an explicit command line for Debian 11+ and the rather comment-less list for other Debian-based systems.
Could this be condensed to something like the following?

> On all Debian-based distributions (...), the required dependencies can be installed with the following command: `...`.
> On Debian 11, you need to run the following addtional command: `...`
> ...

Reply:
I think there's the issue that Debian 10 and 11 need other packages? I'm not happy with this listing at all.

Reply:
For Qt on Debian and Raspbian, both 10 and 11 need `qtbase5-dev qttools5-dev-tools`, and 10 also needs `qt5-default`, which doesn't exist in 11. See the way this was dealt with for Raspbian in https://github.com/jamulussoftware/jamulus/pull/2267#issuecomment-1022127426

Reply:
That sounds very confusing... Can we maybe just tell the user about debian 11 and link your comment for "older versions"?

Reply:
Ok. Shortened now.